tests/dragonfly/replication_test.py

```diff
@@ -70,26 +70,33 @@ async def run_replication(c_replica):
     ), "Weak testcase. Increase number of streamed iterations to surpass full sync"
     await stream_task
 
+    async def check_replica_finished_exec(c_replica):
+        info_stats = await c_replica.execute_command("INFO")
+        tc1 = info_stats['total_commands_processed']
+        await asyncio.sleep(0.1)
+        info_stats = await c_replica.execute_command("INFO")
+        tc2 = info_stats['total_commands_processed']
+        return tc1+1 == tc2 # Replica processed only the info command on above sleep.
+
+    async def check_all_replicas_finished():
+        while True:
+            await asyncio.sleep(1.0)
+            is_finished_arr = await asyncio.gather(*(asyncio.create_task(check_replica_finished_exec(c))
+                                                     for c in c_replicas))
+            if all(is_finished_arr):
+                break
+
```

Comment:
It would be better to make those global functions, but so far we use them only in one test